source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/EverythingBagel.h

```diff
@@ -1,3 +1,3 @@
ock_id\",\"text\":\"Column3_TextBlock_text\",\"type\":\"TextBlock\"}],\"style\":\"Default\",\"type\":\"Column\",\"width\":\"stretch\"}],\"id\":\"ColumnSet_id\",\"separator\":true,\"spacing\":\"large\",\"type\":\"ColumnSet\"}],\"selectAction\":{\"data\":\"Container_data\",\"id\":\"\",\"title\":\"Container_Action.Submit\",\"type\":\"Action.Submit\"},\"spacing\":\"medium\",\"style\":\"Default\",\"type\":\"Container\"},{\"facts\":[{\"title\":\"Topping\",\"value\":\"poppyseeds\"},{\"title\":\"Topping\",\"value\":\"onion flakes\"}],\"id\":\"FactSet_id\",\"type\":\"FactSet\"},{\"id\":\"ImageSet_id\",\"imageSize\":\"Auto\",\"images\":[{\"type\":\"Image\",\"url\":\"https://adaptivecards.io/content/cats/1.png\"},{\"type\":\"Image\",\"url\":\"https://adaptivecards.io/content/cats/2.png\"},{\"type\":\"Image\",\"url\":\"https://adaptivecards.io/content/cats/3.png\"}],\"separator\":true,\"type\":\"ImageSet\"},{\"id\":\"Container_id_inputs\",\"items\":[{\"id\":\"Input.Text_id\",\"inlineAction\":{\"iconUrl\":\"https://adaptivecards.io/content/cats/1.png\",\"id\":\"\",\"title\":\"Input.Text_Action.Submit\",\"type\":\"Action.Submit\"},\"isRequired\":false,\"maxLength\":10,\"placeholder\":\"Input.Text_placeholder\",\"spacing\":\"small\",\"style\":\"text\",\"type\":\"Input.Text\",\"value\":\"Input.Text_value\"},{\"id\":\"Input.Number_id\",\"isRequired\":false,\"max\":10,\"min\":5,\"placeholder\":\"Input.Number_placeholder\",\"type\":\"Input.Number\",\"value\":7},{\"id\":\"Input.Date_id\",\"isRequired\":false,\"max\":\"1/1/2020\",\"min\":\"8/1/2018\",\"placeholder\":\"Input.Date_placeholder\",\"type\":\"Input.Date\",\"value\":\"8/9/2018\"},{\"id\":\"Input.Time_id\",\"isRequired\":false,\"max\":\"17:00\",\"min\":\"10:00\",\"placeholder\":\"Input.Time_placeholder\",\"type\":\"Input.Time\",\"value\":\"13:00\"},{\"id\":\"Input.Toggle_id\",\"isRequired\":false,\"title\":\"Input.Toggle_title\",\"type\":\"Input.Toggle\",\"value\":\"Input.Toggle_on\",\"valueOff\":\"Input.Toggle_off\",\"valueOn\":\"Input.Toggle_on\"},{\"size\":\"Large\",\"text\":\"Everybody's got choices\",\"type\":\"TextBlock\",\"weight\":\"Bolder\"},{\"choices\":[{\"title\":\"Input.Choice1_title\",\"value\":\"Input.Choice1\"},{\"title\":\"Input.Choice2_title\",\"value\":\"Input.Choice2\"},{\"title\":\"Input.Choice3_title\",\"value\":\"Input.Choice3\"},{\"title\":\"Input.Choice4_title\",\"value\":\"Input.Choice4\"}],\"id\":\"Input.ChoiceSet_id\",\"isMultiSelect\":true,\"isRequired\":false,\"style\":\"Compact\",\"type\":\"Input.ChoiceSet\",\"value\":\"Input.Choice2,Input.Choice4\"}],\"type\":\"Container\"},{\"actions\":[{\"id\":\"ActionSet.Action.Submit_id\",\"title\":\"ActionSet.Action.Submit\",\"type\":\"Action.Submit\"},{\"id\":\"ActionSet.Action.OpenUrl_id\",\"title\":\"ActionSet.Action.OpenUrl\",\"type\":\"Action.OpenUrl\",\"url\":\"https://adaptivecards.io/\"}],\"type\":\"ActionSet\"}],\"fallbackText\":\"fallbackText\",\"lang\":\"en\",\"speak\":\"speak\",\"type\":\"AdaptiveCard\",\"version\":\"1.0\"}\n"
```

Comment:
>\"id\":\"\" [](start = 939, length = 11)

The UWP object model tests also need updating.

Reply:
Done :)